README.md

```diff
@@ -59,19 +68,22 @@ See the [contributing document](CONTRIBUTING.md)
 for information on how to contribute and the guidelines for contributions.
 
 ## Initial Committers
-* Manjunath A C (manju956)
-* Daniel Anderson (danintel)
-* Thomas J Barnes (TomBarnes)
-* Srinath Duraisamy (srinathduraisamy)
-* Manoj Gopalakrishnan (manojgop)
-* Karthika Murthy (Karthika)
-* Ramakrishna Srinivasamurthy (ram-srini)
-* Yevgeniy Y. Yarmosh (EugeneYYY)
+
+- Manjunath A C (manju956)
+- Daniel Anderson (danintel)
+- Thomas J Barnes (TomBarnes)
+- Srinath Duraisamy (srinathduraisamy)
+- Manoj Gopalakrishnan (manojgop)
+- Karthika Murthy (Karthika)
+- Ramakrishna Srinivasamurthy (ram-srini)
+- Yevgeniy Y. Yarmosh (EugeneYYY)
 
 ## Sponsor
-* Mic Bowman (cmickeyb) - TSC member
+
+- Mic Bowman (cmickeyb) - TSC member
 
 ## License
+
```

Comment:
Is this a style issue or a markdown validity issue?  If the former is it needed?